docs/rfcs/20210223_external_connectivity_in_k8s.md

```diff
@@ -0,0 +1,230 @@
+- Feature Name: Rafał Korepta
```

Comment:
:))) this made me laugh. Are we implementing Rafal in this PR?

Reply:
Ha, I was distracted. Will fix that :)